### ISSUE TITLE
Use on Android?

### ISSUE THREAD

**Issue by BiLachner** [OpenBoard-org/OpenBoard#101]
I still looking for an App for Android-Devices to use my tablet as whiteboard.

As far as I know, Qt5 software can be used/exported to an Android-App.

Would be really usefull!

**Comment by freeheel3r** [comment]
Hi,
Thanks for your feedback!

Qt does allow building for Android to some extent, but in this case it would require a lot of work. Not everything is fully platform-independent (there is quite a lot of code in OpenBoard that is specific to either Linux, Windows or Mac), and that is without counting the external libraries that OpenBoard requires.

Even if we did have the time to do that, I don't think OpenBoard would work very well on a tablet. The interface would need a lot of work to be usable and to feel like an Android app.

That being said, I did a quick google search and saw many interactive whiteboard apps available for Android -- perhaps one of those would be a good alternative to OpenBoard for now?
